build.sbt

```diff
@@ -121,8 +122,25 @@ def formattingPreferences = {
     .setPreference(SpacesAroundMultiImports, true)
 }
 
+val defaultMultiJvmOptions: List[String] = {
+  import scala.collection.JavaConverters._
+  // multinode.D= and multinode.X= makes it possible to pass arbitrary
+  // -D or -X arguments to the forked jvm, e.g. -Dmultinode.Xmx512m
+  val MultinodeJvmArgs = "multinode\\.(D|X)(.*)".r
+  val knownPrefix = Set("multnode.", "lagom.")
```

Comment:
hm, I copied this from Akka
I think that is different from the ones starting with `multinode`, but it's rather confusing. I'll remove that prefix.